### PR TITLE
Added my virus

### PR DESCRIPTION
**Pull request by yunyicheng**
Forgot to create a PR after completing the assignment ;(

### CODE REVIEW

File: virus/.DS_Store
Comment (ababaian) [inline virus/.DS_Store:1]: Please exclude dot files from your PR.

File: virus/VirususIntricatus Trichomonadis.md
Comment (ababaian) [inline virus/VirususIntricatus Trichomonadis.md:1]: Please don't use spaces ` ` in your filenames. instead use `_`

Or to be consistent with your folder name, just use CamelCase

File: virus/VirususIntricatus Trichomonadis.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+## Results
+
+### Ecology subsection A...
```
Comment (ababaian) [inline virus/VirususIntricatus Trichomonadis.md]: Please use a more informative sub-section title for all sub-sections.

File: virus/VirususIntricatus Trichomonadis.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+### Virus Genome {Q3}
```
Comment (ababaian) [inline virus/VirususIntricatus Trichomonadis.md]: Remove sections if you're not including information in this sub-section please

**Review by ababaian — CHANGES_REQUESTED** [review]

**Comment by yunyicheng** [comment]
@ababaian Hello Prof. Babaian, I have made changes according to your suggestions, thank you!

**Review by ababaian — APPROVED** [review]

**Pull request merged**